src/go/rpk/build.sh

```diff
@@ -14,11 +14,11 @@ img_tag=${version:-latest}
 
 out_dir="$(go env GOOS)-$(go env GOARCH)"
 
-mkdir -p ${out_dir}
+mkdir -p "${out_dir}"
```

Comment:
build.sh quotes should be a separate commit, please split

Reply:
I sent https://github.com/vectorizedio/redpanda/pull/312 for the double quoting in build.sh